web/apps/labelstudio/src/components/Form/Form.scss

```
@@ -7,13 +7,13 @@
     justify-items: stretch;
     justify-content: space-between;
     grid-template-columns: repeat(var(--column-count, 5), 1fr);
-    grid-column-gap: 8px;
-    grid-row-gap: var(--row-gap, 8px);
+    grid-gap: var(--row-gap, 8px) 8px;
 
     &:not(:first-child) {
       margin-top: 12px;
     }
 
+    /* stylelint-disable-next-line selector-type-no-unknown */
     & description {
```

this was an error that was missed - i went through git blame and I see it should be this
```suggestion
    &__description {
```

---

That's make sense